src/node_errors.h

```diff
@@ -19,9 +19,11 @@ void AppendExceptionLine(Environment* env,
                          v8::Local<v8::Message> message,
                          enum ErrorHandlingMode mode);
 
+// This function calls backtrace, it should have not be marked as [[noreturn]].
+// But it is a public API, removing the attribute can break.
 [[noreturn]] void OnFatalError(const char* location, const char* message);
```

Comment:
I think we can just have `InternalOnFatalError` with no `[[noreturn]]` for internal use then keep the public one as `[[noreturn]]` for now (but we can remove it later as a semver-major change), so at least internal call sites can get backtraces correctly.

Reply:
Internally, we can use `UNREACHABLE` instead. Updated internal call sites, except call sites in node-api which exposes `OnFatalError`, and call sites in js-native-api where `util.h` was intentionally not included.

Reply:
Using UNREACHABLE() STGM